src/nnet3/nnet-training.cc

```diff
@@ -235,6 +235,45 @@ void ComputeObjectiveFunction(const GeneralMatrix &supervision,
         computer->AcceptOutputDeriv(output_name, &diff);
       break;
     }
+    case kLognormal: {
```

Comment:
> One of the things that will affect this is whether or not it gives improvements on top of the chain models.

Are there any chain trained models available so I can try ?
@vijayaditya could you please point me to any chain trained models if available ?
Thank you